library/src/CMakeLists.txt

```diff
@@ -76,6 +76,8 @@ if( NOT USE_CUDA )
     if( CUSTOM_ROCBLAS )
       set ( ENV{rocblas_DIR} ${CUSTOM_ROCBLAS})
       find_package( rocblas REQUIRED CONFIG NO_CMAKE_PATH )
+      target_include_directories( hipblas BEFORE PRIVATE ${CUSTOM_ROCBLAS}/include)
```

Comment:
set( ENV ... why wasn't the path just passed into the find_package call ?
Unrelated but just curious.